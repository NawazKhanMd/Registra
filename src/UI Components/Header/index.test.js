import React from 'react';
import renderer from 'react-test-renderer';
import {Header} from './index';
jest.mock('react-router-dom', () => ({
    useHistory: () => ({
      push: jest.fn(),
      location:{pathname:'/'}
    }),
  }));

test('Header Loads', () => {
  const component = renderer.create(
    <Header icon={'Hey'}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});