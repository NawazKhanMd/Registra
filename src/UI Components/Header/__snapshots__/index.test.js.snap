// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Loads 1`] = `
<div
  className="header"
>
  <div
    className="display-flex align-items-center"
  >
    <span
      className="main-link"
    >
      Personal Info
    </span>
  </div>
  <div
    className="actions"
  >
    <span
      className="sub-link"
    >
      User
    </span>
    Hey
  </div>
</div>
`;
